iroha_data_model/src/isi.rs

```diff
@@ -277,6 +359,50 @@ where
     }
 }
 
+#[allow(clippy::len_without_is_empty)]
+impl SetKeyValueBox {
+    /// Calculates number of underneath instructions and expressions
+    pub fn len(&self) -> usize {
+        self.object_id.len() + self.key.len() + self.value.len() + 1
+    }
+
+    /// Default [`SetKeyValueBox`] constructor.
+    pub fn new<
+        I: Into<EvaluatesTo<IdBox>>,
+        K: Into<EvaluatesTo<String>>,
+        V: Into<EvaluatesTo<Value>>,
+    >(
+        object_id: I,
+        key: K,
+        value: V,
+    ) -> Self {
+        Self {
+            object_id: object_id.into(),
+            key: key.into(),
+            value: value.into(),
+        }
+    }
+}
+
+#[allow(clippy::len_without_is_empty)]
+impl RemoveKeyValueBox {
+    /// Calculates number of underneath instructions and expressions
+    pub fn len(&self) -> usize {
+        self.object_id.len() + self.key.len() + 1
+    }
+
+    /// Default [`RemoveKeyValueBox`] constructor.
+    pub fn new<I: Into<EvaluatesTo<IdBox>>, K: Into<EvaluatesTo<String>>>(
```

Comment:
And here it would probably suit better

Reply:
These are the structures I was talking about in the previous comment